CMakeLists.txt

```diff
@@ -1,12 +1,18 @@
 cmake_minimum_required (VERSION 2.8.0)
 project (ibmdb2i)
+######David Lowes - for compilation under IBMi7R2 with gcc from IBM yum repo
 # Dave add -malign-power -malign-natural (below)
 # Our power engine will run better. Also fewer gcc odd boundary over runs.
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -malign-power -malign-natural")
-set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -malign-power -malign-natural")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DMARIA_PLUGIN_INTERFACE_VERSION=536875296 -malign-power -malign-natural -DDBUG_OFF -mlong-double-128")
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DMARIA_PLUGIN_INTERFACE_VERSION=536875296 -malign-power -malign-natural -DDBUG_OFF -mlong-double-128")
```

Comment:
@davidl71, what's the magic you used to determine `-DMARIA_PLUGIN_INTERFACE_VERSION=536875296`?

I believe the other changes to this line are no longer needed due to other source code changes in the master branch